docs/CMakeLists.txt

```diff
@@ -175,14 +179,98 @@ function(build_docs)
                       COMMENT "Pre-process docs"
                       VERBATIM)
 
-    foreach(source_file ${doc_source_files})
+    # ovino doc files
+    file(GLOB_RECURSE ovino_doc_files
+        LIST_DIRECTORIES true RELATIVE ${OpenVINO_MAIN_SOURCE_DIR}
+        "${OpenVINO_MAIN_SOURCE_DIR}/docs/*.md"
+        "${OpenVINO_MAIN_SOURCE_DIR}/docs/*.png"
+        "${OpenVINO_MAIN_SOURCE_DIR}/docs/*.gif"
+        "${OpenVINO_MAIN_SOURCE_DIR}/docs/*.jpg"
+        "${OpenVINO_MAIN_SOURCE_DIR}/inference-engine/*.md"
+        "${OpenVINO_MAIN_SOURCE_DIR}/inference-engine/*.png"
+        "${OpenVINO_MAIN_SOURCE_DIR}/inference-engine/*.gif"
+        "${OpenVINO_MAIN_SOURCE_DIR}/inference-engine/*.jpg")
+
+    foreach(source_file ${ovino_doc_files})
         list(APPEND commands COMMAND ${CMAKE_COMMAND} -E copy
-            "${OpenVINO_MAIN_SOURCE_DIR}/${source_file}" "${DOCS_BUILD_DIR}/${source_file}")
+            "${OpenVINO_MAIN_SOURCE_DIR}/${source_file}" "${DOCS_BUILD_DIR}/openvino/${source_file}")
     endforeach()
 
+    # omz doc files
+    if(EXISTS "${OMZ_DOCS_DIR}")
+        get_filename_component(OMZ_DOCS_DIR "${OMZ_DOCS_DIR}" ABSOLUTE)
+
+        file(GLOB_RECURSE omz_doc_files
+            LIST_DIRECTORIES true RELATIVE ${OMZ_DOCS_DIR}
+            "${OMZ_DOCS_DIR}/*.md"
+            "${OMZ_DOCS_DIR}/*.png"
+            "${OMZ_DOCS_DIR}/*.gif"
+            "${OMZ_DOCS_DIR}/*.jpg")
+
+        foreach(source_file ${omz_doc_files})
+            list(APPEND commands COMMAND ${CMAKE_COMMAND} -E copy
+                "${OMZ_DOCS_DIR}/${source_file}" "${DOCS_BUILD_DIR}/omz/${source_file}")
+        endforeach()
+        configure_file("${OMZ_DOCS_DIR}/omz_docs.xml" "${DOCS_BUILD_DIR}/omz_docs.xml" @ONLY)
+    endif()
+
+    # workbench doc files
+    if(EXISTS "${WORKBENCH_DOCS_DIR}")
```

Comment:
let's add all such variables like `WORKBENCH_DOCS_DIR` as `set(WORKBENCH_DOCS_DIR "" CACHE PATH "Path to WorkBench documentation")`

Reply:
Done